Add details on cloudwatch stats (#16949)
DataDog/documentation: content/en/integrations/guide/aws-integration-and-cloudwatch-faq.md

Some important distinctions to be aware of:

> - Datadog collects a single CloudWatch statistic for the equivalent CloudWatch metric name in Datadog. If you are looking at `Sum` in CloudWatch but Datadog is pulling the `Average` CloudWatch statistic, this will result in discrepencies. For some CloudWatch metrics, multiple statistics can be useful and Datadog creates different metric names for the same CloudWatch metric with different statistics. Ex. `aws.elb.latency` and `aws.elb.latency.maximum`.

Reviewer: ```suggestion
- Datadog collects a single CloudWatch statistic for the equivalent CloudWatch metric name in Datadog. Comparing the `Sum` in CloudWatch to the `Average` in Datadog results in discrepancies. For some CloudWatch metrics, multiple statistics can be useful and Datadog creates different metric names for the same CloudWatch metric with different statistics. For example, `aws.elb.latency` and `aws.elb.latency.maximum`.
```
